spec/lib/pre_assembly/digital_object_spec.rb

```diff
@@ -10,8 +10,12 @@
   let(:bc) { create(:batch_context, staging_location: 'spec/fixtures/images_jp2_tif') }
   let(:druid) { object.druid }
   let(:tmp_dir_args) { [nil, 'tmp'] }
+  let(:tmp_area) do
+    Rails.root.join(Dir.mktmpdir(*tmp_dir_args)).to_s
+  end
+  let(:assembly_directory) { PreAssembly::AssemblyDirectory.new(druid_id: object.druid.id, common_path: tmp_area) }
 
```

Comment:
We want to have the full path to the `tmp_area` (which mimics how users use pre-assembly by providing a full path to their content directory) ... since we now depend on removing the common path to files to preserve the hierarchy.  This also moves this setup so it can be used for all tests.